src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -194,7 +194,10 @@ export const generateNavItems = (): NavItem[] => {
     },
     {
       id: 'operator',
-      enabled: () => CLOUD && quartzMe?.isOperator === true,
+      enabled: () =>
+        CLOUD &&
+        (identity.user.operatorRole === 'read-write' ||
+          identity.user.operatorRole === 'read-only'),
       testID: 'nav-item--operator',
```

Comment:
In identity land, there is no `isOperator`: there is only ~~Zuul~~ `operatorRole`.

`OperatorRole` two potential values - `read-write` or `read-only`. Since this check is what gives access to the operator route, I'd rather an extra degree of safety here to check not just that `operatorRole` is populated, but that it's exactly what we expect it to be - read-write or read-only.